src/smc-webapp/project_page.cjsx

```diff
@@ -258,6 +258,7 @@ FreeProjectWarning = rclass ({name}) ->
         redux.getActions('billing')?.update_customer()
 
     render : ->
+        console.log(JSON.stringify(@state))
```

Comment:
Did you not see that was removed by a follow up commit? Please add the needs review label back.